.stylelintrc.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
         'function-url-quotes': 'always',
         'media-feature-colon-space-after': 'always',
         'media-feature-colon-space-before': 'never',
-        'media-feature-range-operator-space-after': 'always',
-        'media-feature-range-operator-space-before': 'always',
+        'media-feature-range-operator-space-after': 'never',
+        'media-feature-range-operator-space-before': 'never',
```

Comment:
These two rules add extra space to the @if statements which lead to error in sass.